crates/hotshot/src/traits/networking/combined_network.rs

```diff
@@ -411,6 +411,31 @@ impl<TYPES: NodeType> ConnectedNetwork<Message<TYPES>, TYPES::SignatureKey>
         .await
     }
 
+    async fn publish_transaction<VER: StaticVersionType + 'static>(
```

Comment:
If we want, this could be an anyhow and we just map the error from whatever legacy error type.

Reply:
Yeah I think based on our discussion earlier it might be worth evaluating if this is internal or external, and we can do errors depending on what it ends up being.